src/utils/tokens/ethereum-goerli-tokens.json

```diff
@@ -9,154 +9,130 @@
   },
   {
     "chain": "ethereum",
-    "address": "0xd3A691C852CDB01E281545A27064741F0B7f6825",
-    "name": "Wrapped BTC",
-    "symbol": "WBTC",
+    "address": "0xA253295eC2157B8b69C44b2cb35360016DAa25b1",
+    "name": "Compound 0x",
+    "symbol": "cZRX",
     "decimals": 8,
-    "iconUrl": ""
+    "iconUrl": "https://raw.githubusercontent.com/compound-finance/token-list/master/assets/ctoken_zrx.svg"
   },
   {
     "chain": "ethereum",
-    "address": "0x50DD65531676F718B018De3dc48F92B53D756996",
-    "name": "Reputation",
-    "symbol": "REP",
-    "decimals": 18,
-    "iconUrl": "https://raw.githubusercontent.com/compound-finance/token-list/master/assets/asset_REP.svg"
+    "address": "0xD87Ba7A50B2E7E660f678A895E4B72E7CB4CCd9C",
+    "name": "USD Coin USDC",
+    "symbol": "USDC",
+    "decimals": 6,
+    "iconUrl": "https://raw.githubusercontent.com/compound-finance/token-list/master/assets/asset_USDC.svg"
   },
   {
     "chain": "ethereum",
-    "address": "0xAf45ae737514C8427D373D50Cd979a242eC59e5a",
-    "name": "Compound 0x",
-    "symbol": "cZRX",
+    "address": "0x1d70B01A2C3e3B2e56FcdcEfe50d5c5d70109a5D",
+    "name": "Compound Augur",
+    "symbol": "cREP",
     "decimals": 8,
-    "iconUrl": "https://raw.githubusercontent.com/compound-finance/token-list/master/assets/ctoken_zrx.svg"
+    "iconUrl": "https://raw.githubusercontent.com/compound-finance/token-list/master/assets/ctoken_rep.svg"
   },
   {
     "chain": "ethereum",
-    "address": "0xa1fAA15655B0e7b6B6470ED3d096390e6aD93Abb",
-    "name": "Compound Wrapped BTC",
-    "symbol": "cWBTC",
-    "decimals": 8,
-    "iconUrl": "https://raw.githubusercontent.com/compound-finance/token-list/master/assets/ctoken_wbtc.svg"
+    "address": "0x183Faf58c4461972765f3F90c6272A4ecE66Bd96",
+    "name": "Reputation",
+    "symbol": "REP",
+    "decimals": 18,
+    "iconUrl": "https://raw.githubusercontent.com/compound-finance/token-list/master/assets/asset_REP.svg"
   },
   {
     "chain": "ethereum",
-    "address": "0xb7a4F3E9097C08dA09517b5aB877F7a917224ede",
-    "name": "USD Coin USDC",
-    "symbol": "USDC",
-    "decimals": 6,
-    "iconUrl": "https://raw.githubusercontent.com/compound-finance/token-list/master/assets/asset_USDC.svg"
+    "address": "0x5D4373F8C1AF21C391aD7eC755762D8dD3CCA809",
+    "name": "Compound Sai",
+    "symbol": "cSAI",
+    "decimals": 8,
+    "iconUrl": "https://raw.githubusercontent.com/compound-finance/token-list/master/assets/ctoken_sai.svg"
   },
   {
     "chain": "ethereum",
-    "address": "0x9de9cde889A40B7402D824386064D17792298E1B",
-    "name": "Pillar",
-    "symbol": "PLR",
+    "address": "0x8e9192D6f9d903b1BEb3836F52a9f71E05846e42",
+    "name": "Sai Stablecoin v1.0",
+    "symbol": "SAI",
     "decimals": 18,
-    "iconUrl": "https://assets.coingecko.com/coins/images/809/thumb/PLR_Token.png?1595237935"
+    "iconUrl": "https://raw.githubusercontent.com/compound-finance/token-list/master/assets/asset_SAI.svg"
   }
 ]
```

Comment:
New line for consistency

Reply:
Goerli Tokens fetched from Etherspot Playground. here need any changes?

Reply:
Please see the above screenshot, a new line is needed at the end of the file

Reply:
https://github.com/pillarwallet/pillarwallet/pull/3174/commits/6f395eb16feb5a46f52e7f8a0be34fe304557c21